api/envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto

```diff
@@ -238,6 +238,52 @@ message HttpConnectionManager {
     google.protobuf.BoolValue enabled = 3;
   }
 
+  // Transformations that apply to path headers. The policy to apply these
+  // transformations can be tuned by the configuration field. Transformations
+  // can be applied internally only, so that the normalized header is used in
+  // matching, routing, and HTTP filters while the original path header is
+  // forwarded upstream, or additionally apply to the forwarded *:path* header.
+  message PathNormalizationOptions {
+    // Defines a policy to apply normalization.  Path rewrites :ref:`regex_rewrite
+    // <envoy_api_field_route.RouteAction.regex_rewrite>` or :ref:`prefix_rewrite
+    // <envoy_api_field_route.RouteAction.prefix_rewrite>` will apply to the
+    // *:path* header destined for the upstream.
+    enum NormalizationPolicy {
+      // Normalizations apply internally before any processing of requests by HTTP
+      // filters, routing, and matching, *and* will affect the upstream *:path* header
+      // as well.
+      FORWARD = 0;
```

Comment:
Should `FORWARD` be the default behavior?
I'm not sure what the intended default should be, and in this case it might make more sense to add a new value `POLICY_UNSPECIFIED = 0` if the policy should be explicit, and increase the values of `FORWARD` and `INTERNAL` by 1.

Reply:
Just saw in the doc update that `FORWARD` is the intended default behavior.